test/cocotb/osdtestlib/modtestclasses.py

```diff
@@ -0,0 +1,637 @@
+import cocotb
+from cocotb.triggers import Timer
+from cocotb.clock import Clock
+from cocotb.result import TestFailure
+from cocotb.triggers import RisingEdge
+
+class NoC_Driver:
```

Comment:
Please use a camel-case name for the class (https://www.python.org/dev/peps/pep-0008/#class-names).